tests/cases/compiler/argumentsReferenceInObjectLiteral_Js.ts

```diff
@@ -0,0 +1,18 @@
+// @checkJs: true
+// @allowJs: true
+// @target: es6
+// @outDir: ./out
+// @filename: /a.js
+
+const a = () => {
+    return {
+        arguments: [],
+    };
+};
+
+const b = () => {
+    const c = {
+        arguments: [],
+    }
+    return c;
+};

```

Comment:
Can you add a test that uses shorthand property assignments?

```ts
const a = () => {
    return {
        arguments,
    };
};
```

Reply:
Do I need to define `arguments` in upper scope?

Reply:
I've added two additional cases.